test/recipes/30-test_evp_data/evpkdf_tls12_prf.txt

```diff
@@ -58,6 +58,7 @@ Ctrl.label = seed:master secret
 Ctrl.client_random = hexseed:36c129d01a3200894b9179faac589d9835d58775f9b5ea3587cb8fd0364cae8c
 Ctrl.server_random = hexseed:f6c9575ed7ddd73e1f7d16eca115415812a43c2b747daaaae043abfb50053fce
 Result = KDF_DERIVE_ERROR
+Reason = invalid key length
```

Comment:
This test is testing the wrong thing..

Reply:
It does appear to be wrong but the seed is, in fact, too short.
I don't think this is the proper place to "fix" this, a new issue/PR would be.

Reply:
Issue raised: #25137